src/app/core/services/product-details-page.service.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ProductDetailsPageService } from './product-details-page.service';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpClientModule } from '@angular/common/http';
```

Comment:
### Breaking the Styleguide detected

Separete "**Theirs** first from **Ours**". Aplly this pattern for all files

#### Example how should be
https://angular.io/guide/styleguide#import-line-spacing

![screen shot 2019-01-31 at 15 15 17](https://user-images.githubusercontent.com/9499109/52056575-4ae8a600-256b-11e9-8ece-e091e835e988.png)